client/my-sites/patterns/controller.tsx

```diff
@@ -1,16 +1,39 @@
+import wpcom from 'calypso/lib/wp';
 import Patterns from 'calypso/my-sites/patterns/patterns';
 import { getCurrentUserLocale } from 'calypso/state/current-user/selectors';
 import type { Context as PageJSContext } from '@automattic/calypso-router';
 
 type Next = ( error?: Error ) => void;
 
 export function fetchPatterns( context: PageJSContext, next: Next ) {
-	const locale = getCurrentUserLocale( context.store.getState() ) || context.lang || 'en';
+	const { queryClient, lang, store, cachedMarkup } = context;
 
-	// eslint-disable-next-line no-console
-	console.log( '############', locale );
+	if ( cachedMarkup ) {
+		next();
 
-	next();
+		return;
+	}
```

Comment:
Do we need this here when we also have this?

https://github.com/Automattic/wp-calypso/blob/e1f240e1fb5a4083592a787b3e6a4f4c9f553692/client/controller/shared.js#L22-L25

Reply:
Right! 🤦‍♂️ Makes sense